docs/BreakingChanges/110 NetDataContractSerializer throws an exception while trying to de-serialize a ConcurrentDictionary.md

```diff
@@ -0,0 +1,38 @@
+## 110: NetDataContractSerializer throws an exception while trying to de-serialize a ConcurrentDictionary serialized with the .NET Framework 4.5
```

Comment:
{
  "Id": "110",
  "Title": "NetDataContractSerializer throws an exception while trying to de-serialize a ConcurrentDictionary serialized with the .NET Framework 4.5",
  "Details": "By design, the [NetDataContractSerializer](https://msdn.microsoft.com/en-us/library/system.runtime.serialization.netdatacontractserializer.aspx) can be used only if both the serializing and deserializing ends share the same CLR types. Therefore, it is not guaranteed that an object serialized with one version of the .NET Framework can be deserialized by a different version.\n\nConcurrentDictionary is a type that is known to not to deserialize correctly if serialized with the .NET Framework 4.5 or earlier and deserialized with the .NET Framework 4.5.1 or later.",
  "Suggestion": "There are a number of possible work-arounds for this issue:\n\n- Upgrade the serializing computer to use the .NET Framework 4.5.1, as well.\n- Use DataContractSerializer instead of NetDataContractSerializer as this does not expect the exact same CLR types at both serializing and deserializing ends.\n- Use Dictionary instead of ConcurrentDictionary since it does not exhibit this particular 4.5->4.5.1 break.",
  "Link": "http://stackoverflow.com/questions/19585791/net-4-5-1-wcf-serialization-exception",
  "Markdown": null,
  "ApplicableApis": [
    "M:System.Runtime.Serialization.NetDataContractSerializer.Deserialize(System.IO.Stream)"
  ],
  "Related": null,
  "VersionBroken": "4.5.1",
  "VersionFixed": null,
  "IsBuildTime": false,
  "IsQuirked": false,
  "IsSourceAnalyzerAvailable": false,
  "BugLink": "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/812565",
  "Notes": "Should be pretty straightforward for an analyzer to check that ConcurrentDictionaries aren't being serialized by NetDataContractSerializers\nSource analyzer status: Pri 2, Planned",
  "IsRetargeting": false,
  "ImpactScope": 2
}

**Parsed APIs:**
- [x] M:System.Runtime.Serialization.NetDataContractSerializer.Deserialize(System.IO.Stream)